pyvista/core/filters/data_set.py

```diff
@@ -2821,30 +3325,33 @@ def extract_feature_edges(dataset, feature_angle=30, boundary_edges=True,
 
         If the given mesh is not PolyData, the external surface of the given
         mesh is extracted and used.
-        From vtk documentation, the edges are one of the following
 
-            1) boundary (used by one polygon) or a line cell
-            2) non-manifold (used by three or more polygons)
-            3) feature edges (edges used by two triangles and whose
+        From vtk documentation, the edges are one of the following:
+
+            1) Boundary (used by one polygon) or a line cell
+            2) Non-manifold (used by three or more polygons)
+            3) Feature edges (edges used by two triangles and whose
                dihedral angle > feature_angle)
-            4) manifold edges (edges used by exactly two polygons).
+            4) Manifold edges (edges used by exactly two polygons).
 
         Parameters
         ----------
         feature_angle : float, optional
-            Defaults to 30 degrees.
+            Feature angle used to detect sharp edges on the mesh.
+            Valid only when ``feature_edges=True``.  Defaults to 30
+            degrees.
 
         boundary_edges : bool, optional
-            Defaults to True
+            Extract the boundary edges. Defaults to ``True``.
 
         non_manifold_edges : bool, optional
-            Defaults to True
+            Extract non-manifold edges. Defaults to ``True``
 
         feature_edges : bool, optional
-            Defaults to True
+            Extract edgs exceeding ``feature_angle``.  Defaults to ``True``
 
         manifold_edges : bool, optional
-            Defaults to True
+            Extract manifold edges. Defaults to ``True``.
 
         inplace : bool, optional
             Updates existing dataset with the extracted features.
```

Comment:
Is my hunch correct that when the input is not `PolyData` (say, an `UnstructuredGrid`) then `inplace=True` doesn't change the input?

Reply:
Correct.  This is broken and I'm removing this...